### ISSUE TITLE
Error: aBuy and iBuy need chest - when attached to chest.

### ISSUE THREAD

**Issue by CoverArtist** [TheoKah/CarrotShop#71]
I've tried both aBuy/iBuy, permissions are set properly, area is in spawn, I'm OPed and items are in chest- but it wont set up shop. Keeps saying it needs a chest (using vanilla minecraft chest)

**Comment by TheoKah** [comment]
Are you following the wiki?
https://github.com/TheoKah/CarrotShop/wiki

What message do you have when you hit the chest with redstone?

**Comment by CoverArtist** [comment]
Yes I'm following it to the T, it says "(ibuy/abuy) needs a chest" but its directly on a chest.

**Comment by CoverArtist** [comment]
![image](https://user-images.githubusercontent.com/5324551/65366312-c1fca880-dbe7-11e9-886f-8d729005463d.png)

**Comment by TheoKah** [comment]
Follow the wiki: click the chest before you click the sign

**Comment by TheoKah** [comment]
Is this still a problem?
